client/src/views/CellView.tsx

```diff
@@ -66,6 +67,26 @@ const CellView = () => {
     });
   }, [Alert, Confirm, brute, navigate, t, updateData]);
 
+  const switchBrute = useCallback((side : number) => {
+    if (user && brute && user.brutes) {
+      const index = user.brutes.findIndex((bruteTemp) => bruteTemp.id === brute.id) + side;
+      // handle the case of index = -1
+      if (index === -1 && side < 0) {
+        navigate(`/${user.brutes[user.brutes.length - 1].name}/cell`);
+      } else {
+        navigate(`/${user.brutes[index % user.brutes.length].name}/cell`);
+      }
+    }
+  }, [brute, navigate, user]);
```

Comment:
There should never be a case with a negative index, as the switch should only be accessible on your own brutes.

```suggestion
  const switchBrute = useCallback((side : number) => {
    if (!brute || !user.brutes) return;
    
    const currentBruteIndex = user.brutes.findIndex((bruteTemp) => bruteTemp.id === brute.id);
    if (currentBruteIndex < 0) return;
    
    const newBrute = user.brutes[(currentBruteIndex + side) % user.brutes.length];
    if (!newBrute) return;
    
    navigate(`/${newBrute.name}/cell`);
  }, [brute, navigate, user]);
```